### ISSUE TITLE
Maßnahme: Von zu Hause aus arbeiten

### ISSUE THREAD

**Issue by AnnKRink** [floAr/WirVsVirus_Simulator#21]
**Maßnahme: Von zu Hause aus arbeiten**

**Problem:** Genau wie Schulen, Kitas und vielen Freizeiteinrichtungen sind auch die Arbeitsstätten der meisten Menschen Orte, an denen sich das neue Coronavirus rasant verbreiten kann. Hinzu kommt, dass in Deutschland sehr viele Menschen mit öffentlichen Verkehrsmitteln zur Arbeit und nach Hause fahren – auch das ist ein weiterer Risikofaktor. 

**Lösung:** Viele Aufgaben, die Beschäftigte auf der Arbeit erledigen, lassen sich auch zu Hause erledigen. Das gilt vor allem für klassische Bürojobs. Es gibt mittlerweile viele technische Möglichkeiten, dass Menschen theoretisch von überall aus gut arbeiten können. In vielen Unternehmen ist das sogenannte Homeoffice sowieso schon üblich - um die Ausbreitung von Corona zu verhindern, kann das mobile Arbeiten aber noch deutlich ausgebaut werden. Dazu können Arbeitgeber beispielsweise mehr Beschäftigte mit Laptops oder Smartphones ausstatten oder ihnen Programme für Telefon- oder Videokonferenzen installieren. Natürlich ist das nicht in allen Berufen möglich.

**Fazit:** Je mehr Menschen von zu Hause aus arbeiten, anstatt zur Arbeit zu fahren, desto langsamer verbreitet sich das Virus. Gleichzeitig wird es für viele Eltern leichter, ihre Kinder zu betreuen, falls diese nicht mehr in die Schule oder die Kita gehen können.

**Comment by AlpixTM** [comment]
Können wir die Maßnahme bitte vereinfacht Homeoffice nennen dort wo man toggelt?
Im Info Text dann gerne "Von zu Hause aus arbeiten"

**Comment by AnnKRink** [comment]
Klar. Ich dachte, Homeoffice ist vielleicht wieder zu sehr ein Begriff aus der Akademiker-Büromenschen-Bubble. Aber ich glaube, der ist mittlerweile medial auch präsent genug, dass man ihn hier verwenden kann.

**Comment by AlpixTM** [comment]
Sollte fürs UI immer möglichst kurz sein denke ich, das ist der Gedanke, der mich treibt. :)

**Comment by AlpixTM** [comment]
![image](https://user-images.githubusercontent.com/17622853/77235718-7cca2b00-6bb8-11ea-98c1-3c1684fe0b5f.png)
Haben jetzt einen Mix

**Comment by AnnKRink** [comment]
Find ich gut!

**Comment by twiggiHH** [comment]
Es gibt ja sogar Werksschliessungen und Ladenschliessungen. Die sollten ggf. als eigene Maßnahme behandelt werden. Eine Werksschlissung hat noch am ehesten den Effekt wie ein HomeOffice. Während Ladenschließungen zusätzlich auch zu weniger Kontaktmöglichkeiten bei den Kunden führen.
